src/Algebra/Bundles.agda

```diff
@@ -670,6 +670,25 @@ record CommutativeSemiring c ℓ : Set (suc (c ⊔ ℓ)) where
     }
 
 
+record CommutativeCancellativeSemiring c ℓ : Set (suc (c ⊔ ℓ)) where
```

Comment:
Looking at the example of `IdempotentCommutativeMonoid`, if you're adding the cancellative property on top of the commutative semiring properties, this should really be called `CancellativeCommutativeSemiring` instead of `CommutativeCancellativeSemiring`. The same goes for the structure as well.